eclipse-scout-core/src/tile/TileGrid.ts

```diff
@@ -1574,12 +1616,12 @@ export class TileGrid extends Widget implements TileGridModel {
    * If virtual is false, the live list of filtered tiles is returned, because every tile has to be rendered.
    * If virtual is true, the rendered tiles are collected and returned.
    */
-  renderedTiles(): Tile[] {
+  renderedTiles(): (TTile | PlaceholderTile)[] {
```

Comment:
Do you really want to return placeholders here? Are they of any interest to developers? Maybe they should be completely hidden for them and therefore also not be part of the public API?